pkg/controller/account/byoc.go

```diff
@@ -201,7 +201,7 @@ func CreateRole(reqLogger logr.Logger, byocRole string, principalARN string, byo
 		RoleName:                 aws.String(byocRole),
 		Description:              aws.String("AdminAccess for BYOC"),
 		AssumeRolePolicyDocument: aws.String(string(jsonAssumeRolePolicyDoc)),
-		Tags: tags,
+		Tags:                     tags,
```

Comment:
This somehow was getting caught in CI gofmt check.

Reply:
This stupid issue. The gofmt -s rules changed in a go minor version update.

Reply:
`gvm use go1.10` to make this run locally the same as it does in CI